### ISSUE TITLE
Uninformative logging during training

### ISSUE THREAD

**Issue by NikZak** [google/automl#697]
During training I see lines that don't have a lot of useful information in my view 
INFO:tensorflow:examples/sec: 3.72883
I0819 18:25:37.922217 140171346902080 tpu_estimator.py:2351] examples/sec: 3.72883
INFO:tensorflow:global_step/sec: 0.954079
I0819 18:25:38.970135 140171346902080 tpu_estimator.py:2350] global_step/sec: 0.954079
INFO:tensorflow:examples/sec: 3.81631
I0819 18:25:38.970345 140171346902080 tpu_estimator.py:2351] examples/sec: 3.81631

If training goes for a long time it would be useful to see at what step are we and what is the current training loss.
I suggest adding smth along following lines in [det_model_fn.py](https://github.com/google/automl/blob/master/efficientdet/det_model_fn.py#L373):
```python
    logging_hook = tf.train.LoggingTensorHook(
        {
            "step": global_step,
            "det_loss": det_loss,
            "cls_loss": cls_loss,
            "box_loss": box_loss,
        },
        every_n_iter=50,
    )
    training_hooks.append(logging_hook)
```

**Comment by kartik4949** [comment]
@NikZak Sure ,let me look.
thanks for insights.

**Comment by mingxingtan** [comment]
Thank you @NikZak   Fixed now!
